std/hash/sha3_test.ts

```diff
@@ -6,9 +6,13 @@ import {
   Keccak256,
   Keccak384,
   Keccak512,
+  // deno-lint-ignore camelcase
   Sha3_224,
+  // deno-lint-ignore camelcase
   Sha3_256,
+  // deno-lint-ignore camelcase
   Sha3_384,
+  // deno-lint-ignore camelcase
```

Comment:
This is not ideal and will pop up in user code as well, discussed in: https://github.com/denoland/deno/issues/7739
We're suppose to remove those modules anyway